server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -554,18 +556,17 @@ public void testCopyToWithWhereNoMatch() throws Exception {
 
     @Test
     public void testCopyFromNestedArrayRow() throws Exception {
-        // assert that rows with nested arrays aren't imported
         execute("create table users (id int, " +
             "name string) with (number_of_replicas=0)");
         execute("copy users from ? with (shared=true)", new Object[]{
             nestedArrayCopyFilePath + "nested_array_copy_from.json"});
-        assertEquals(1L, response.rowCount()); // only 1 document got inserted
+        assertEquals(2L, response.rowCount());
         refresh();
 
-        execute("select * from users");
-        assertThat(response.rowCount(), is(1L));
+        execute("select * from users order by id");
+        assertThat(response.rowCount()).isEqualTo(2L);
 
-        assertThat(printedTable(response.rows()), is("2| Trillian\n"));
+        assertThat(response).hasRows("1| Ford", "2| Trillian");
     }
```

Comment:
On master we have only 1 row imported, I guess because entry with non-existent in table column "nested" fails validation.

Current behavior of JSON:
use table columns as target columns, ignore other columns. No dynamic mapping updates since targetCols=tableCols. This is similar to handling of CSV with no header and no target cols.

Another possible handling:
 Treat json in a same way as CSV with `header = true` and `targetCols = empty`, ie this test should fail with default `strict` policy and should become dynamic mapping update with `dynamic` policy. That would be a breaking change because of strict tables/new columns. Wdyt?